modules/metastorage/src/test/java/org/apache/ignite/internal/metastorage/watch/WatchAggregatorTest.java

```diff
@@ -274,8 +274,8 @@ private Entry entry(String key, String val, long revision, long updateCntr) {
         return new Entry() {
             /** {@inheritDoc} */
             @Override
-            public @NotNull ByteArray key() {
-                return new ByteArray(key);
+            public @NotNull byte[] key() {
```

Comment:
Let's remove `@NotNull`

Reply:
This class will be removed altogether in my next PR, so I don't think it's necessary